protocol/meson.build

```diff
@@ -17,6 +17,8 @@ wayland_scanner_client = generator(
 
 client_protocols = [
 	wl_protocol_dir / 'stable/xdg-shell/xdg-shell.xml',
+	wl_protocol_dir / 'staging/cursor-shape/cursor-shape-v1.xml',
+	wl_protocol_dir / 'unstable/tablet/tablet-unstable-v2.xml',
```

Comment:
Nit: we don't need this protocol

Reply:
A linking error occurs without it.

Reply:
Ah, right. Probably an access to the `zwp_tablet_tool_v2_interface` global variable.